src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -324,23 +324,40 @@ export default function ItemDetailPage(props: Props) {
                   <div className="title">{t('item_detail_page.selling.title')}</div>
                   <div className="data">
                     {isFree(item) ? (
-                      <Section>
+                      <Section className="price-container">
                         <div className="subtitle">{t('item.price')}</div>
                         <div className="value">{t('global.free')}</div>
                       </Section>
                     ) : item.price ? (
-                      <Section>
+                      <Section className="price-container">
```

Comment:
The price container seems to have broken how the price looks:
<img width="690" height="229" alt="Screenshot 2025-07-22 at 1 21 05 PM" src="https://github.com/user-attachments/assets/20b3c88d-9b43-4e99-bf89-0024251d4bb9" />
Would you mind checking it out?